README.rst

```diff
@@ -939,6 +1009,194 @@ configuration status of a device changes, this ensures the check reacts
+
+.. image:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/enable-openwrt-ssh.png
+  :alt: Enable ssh access from openwisp to device
+  :align: center
+  
+3. Configure Iperf settings
+###########################
+
+Configure iperf servers in `openwisp settings <https://github.com/openwisp/openwisp-monitoring/blob/master/tests/openwisp2/settings.py>`_ , 
+The host can be specified by hostname, IPv4 literal, or IPv6 literal.
+
+For example.
+
+.. code-block:: python
+
+    OPENWISP_MONITORING_IPERF_SERVERS = {
+    # Public iperf servers also available: https://iperf.fr/iperf-servers.php#public-servers
+    # '<org-pk>': ['<ORG_IPERF_SERVER>']
+    'a9734710-db30-46b0-a2fc-01f01046fe4f': ['speedtest.uztelecom.uz'],
+    'z9734710-db30-46b0-a2fc-01f01046fe4f': ['192.168.5.109'],
+    'c9734710-db30-46b0-a2fc-01f01046fe4f': ['2001:db8::1'],
+    }
+
+Add celery beat configuration for iperf check in `openwisp settings 
+<https://github.com/openwisp/openwisp-monitoring/blob/master/tests/openwisp2/settings.py>`_
+
+.. code-block:: python
+
+    CELERY_BEAT_SCHEDULE = {
+        # Other celery beat configurations
+
+        # Celery beat configuration for iperf check
+        'run_iperf_checks': {
+            'task': 'openwisp_monitoring.check.tasks.run_checks',
+            # https://docs.celeryq.dev/en/latest/userguide/periodic-tasks.html#crontab-schedules
+            # Executes check every 5 mins from 00:00 AM to 6:00 AM (night)
+            'schedule': crontab(minute='*/5', hour='0-6'),
+            # Iperf check path
+            'args': (['openwisp_monitoring.check.classes.Iperf'],),
+            'relative': True,
+        }
+    }
+
+**Note:** We recommended to configure this check for night or during non peak traffic times to not interfere with standard traffic.
+
+4. Run the check
+################
+
+This should happen automatically if you have celery running in the background. For testing, you can
+run this check manually using the `run_checks <#run_checks>`_ command. After that, you should see the
+iperf network measurements charts.
+
+.. image:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/iperf-charts.png
+  :alt: Iperf network measurement charts
+
+
+Configure Iperf check for authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default iperf check runs without any kind of **authentication**, But we can configure it to use **RSA authentication** 
+between the **client** and the **server** to restrict the connections to the server & only allow legitimate clients.
+
+At Iperf server
+###############
+
+1. Generate RSA keypair
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: shell
+
+   openssl genrsa -des3 -out private.pem 2048
+   openssl rsa -in private.pem -outform PEM -pubout -out public.pem
+   openssl rsa -in private.pem -out private_not_protected.pem -outform PEM
+
+After running above mention commands, the public key will be contained in the 
```

Comment:
```suggestion
After running the commands mentioned above, the public key will be stored in ``public.pem`` 
```

Keep it short, apply a similar change also to ``private_not_protected.pem``, I would also rename that as `private_key.pem`, while I would rename `public.pem` to `public_key.pem`.

Reply:
Sure :)